docs/pre-release/reference/intercepts/index.md

```diff
@@ -68,14 +68,25 @@ your cluster; if it detects it correctly, may simply press "enter" and
 accept the default, otherwise you must tell Telepresence the correct
 value.
 
-When creating an intercept with the `http` mechanism, the
-traffic-agent sends a `GET /telepresence-http2-check` request to your
-service and to the process running on your local machine at the port
-specified in your intercept, in order to determine if they support
-HTTP/2.  This is required for the intercepts to behave correctly.  If
-you do not have a service running locally when the intercept is
-created, the traffic-agent will use the result it got from checking
-the in-cluster service.
+When creating an intercept with the `http` mechanism, Telepresence must
+determine the application protocol to use (HTTP/1.1 or HTTP/2). If the
+service's `ports.appProtocol` field is set, it will be used. If not, then
+Telepresence will use the configured [application protocol strategy](../config/#intercept)
+to determine the protocol. The default behavior (`http2Probe` strategy) is
+to send a `GET /telepresence-http2-check` request to your service in order
+to determine if it supports HTTP/2.  This is required for the intercepts to
+behave correctly.
+
+### TLS
+
+If the intercepted service expects TLS, then Telepresence must terminate the
+TLS in order to use the `http` mechanism. It is then common to also restore
+the TLS by originating it again. This is controlled using [TLS annotations](../cluster-config/#tls)
+in the workload.
```

Comment:
Just trimmed down the verbiage a bit:
_If the intercepted service expects TLS, then Telepresence terminates the TLS to use the http mechanism. It is then common to also restore the TLS by originating it again. This is controlled using TLS annotations in the workload._

Quick question: "It is then common to also restore the TLS by originating it again" seems like a potentially confusing statement. Is that done automatically? If so, it's worth mentioning.

Reply:
Yeah it's not done automatically, which is why the original linked to the TLS annotations section bc users need to follow the instructions there (e.g. adding annotations) to their workload. So should we add that back in or make it more clear that they actually need to go to that page and do what the docs there suggest?

Reply:
oh whoops I hadn't removed the first version of this from the file. So thanks for calling my attention to this. that has been fixed!